loopchain/baseservice/node_subscriber.py

```diff
@@ -132,13 +134,13 @@ def __del__(self):
         await self._websocket.send(json.dumps(request))
 
     async def _recv_until_timeout(self):
-        response: bytes = await asyncio.wait_for(
+        response: str = await asyncio.wait_for(
             fut=self._websocket.recv(),
             timeout=2 * conf.TIMEOUT_FOR_WS_HEARTBEAT
         )
-        response_dict = convert_response_to_dict(response)
+        response = validate_response(response)
 
-        await ws_methods.dispatch(response_dict)
+        await async_dispatch(response, ws_methods)
```

Comment:
`methods` parameter not passed, `dispatch` use global_methods.
https://github.com/bcb/jsonrpcserver/blob/4.1.2/jsonrpcserver/async_dispatcher.py#L91

Reply:
I passes `methods` parameter to `dispatch` method.

The second positional argument `ws_methods` is the `methods` parameter.

Reply:
I thought that `global_methods` is usable in this case.

Reply:
Keeping scope small is better in most cases if there is no reason to use it globally. 
Can you explain "can't replace `add`" in detail?

Reply:
I means below code in `__init__`.
```py
        ws_methods.add(self.node_ws_PublishHeartbeat)
        ws_methods.add(self.node_ws_PublishNewBlock)
```
So, It's not easy to remove `ws_methods` parameter.
And I already approved.